### ISSUE TITLE
[BITV] 9.1.1.1a/1.1 - The button to display the password in clear text does not have an alternative text as an empty aria-label attribute is used. (1)

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36707]
- [x] https://github.com/nextcloud/nextcloud-vue/pull/3710
- [x] Release - 7.6.0
- [x] https://github.com/nextcloud/server/pull/36813
- [x] https://github.com/nextcloud/server/pull/36841

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v1-n1

https://try.nextcloud.com/ltd/a11y/index.php/login
![image](https://user-images.githubusercontent.com/24800714/218807314-0406c107-6221-460d-bd8a-5823385a2844.png)

</details>
